include/usgscsm/Utilities.h

```diff
@@ -69,6 +69,14 @@ void lagrangeInterp (
   const int&     i_order,
   double*        valueVector);
 
+// Brent's algorithm for finding the roots of a function
+// Arguments are two inputs that bracket a root, the function, and a convergence tolerance
+double brentRoot(
```

Comment:
Keeping comment is fine, but do we really want the function named after someone? :-)

Reply:
It's the name of the algorithm, so I don't see a reason to change it.

Reply:
haha - I thought it was our Brent and he was simply pointing to a generic root finding method.

Reply:
@thareUSGS At first, I thought that, toO!